src/Jenssegers/Mongodb/Schema/Blueprint.php

```diff
@@ -236,14 +236,14 @@ public function expire($columns, $seconds)
     /**
      * @inheritdoc
```

Comment:
Maybe replace doc block on actual?

Reply:
I've added correct doc block, thank you! 👍